service/src/main/java/bio/terra/drshub/services/RestTemplateFactory.java

```diff
@@ -0,0 +1,62 @@
+package bio.terra.drshub.services;
+
+import bio.terra.drshub.config.MTlsConfig;
+import lombok.extern.slf4j.Slf4j;
+import nl.altindag.ssl.SSLFactory;
+import nl.altindag.ssl.util.Apache4SslUtils;
+import nl.altindag.ssl.util.PemUtils;
+import org.apache.http.config.Registry;
+import org.apache.http.config.RegistryBuilder;
+import org.apache.http.conn.socket.ConnectionSocketFactory;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+@Slf4j
```

Comment:
This class doesn't log so it doesn't need this annotation. But maybe the convention here is to always use it?